common/djangoapps/util/tests/test_password_policy_validators.py

```diff
@@ -66,14 +64,11 @@ def test_password_unicode_normalization(self):
         # s ̣ ̇ (s with combining dot below and combining dot above)
         not_normalized_password = u'\u0073\u0323\u0307'
         self.assertEqual(len(not_normalized_password), 3)
-        # When the flag is not set, the validation should succeed since len > 2
-        self.validation_errors_checker(not_normalized_password, None)
```

Comment:
If you still want this part of the test to exist, I can import django's validate password and show how it fails if we only call that without normalizing. Just feels dirtier to me

Reply:
Naw, this as written is fine.